### DISCUSSION TITLE
github Permission denied (publickey), but only for one respository

### DISCUSSION THREAD

**Discussion by justinblayney** [community/community#43277] [Repositories]
### Select Topic Area

Bug

### Body

I have many repositories(sites) within my github account, I used 2 different ones today, making branches, pushing merging..

I went to a third repository today (still within my git account like other 2). and I get

```
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

The repository exists

I used 2 other from same account with last hour and still can even now. They all have exact same permissions

Why is only one site giving me this error and how can I fix?

I tried cloning it fresh, but it wont let me clone

I have pushed to this repository countless times in the past

**Comment by CodesdaLu** [comment]
Hi, 
Error: Permission denied (publickey)

A "Permission denied" error means that the server rejected your connection. There could be several reasons why, and the most common examples are explained below.

link: https://docs.github.com/en/github-ae@latest/authentication/troubleshooting-ssh/error-permission-denied-publickey

I hope, help you. 

if your question has been answered mark answered

**Comment by syedAmeenuddin** [comment]
Authenticate again by using this below commands.
-> gh auth login
-> then, select (SSH) option
-> and then check by this cmd: ssh -T git@github.com

issue will be solved.... it worked on my MAC..

**Comment by Bartletti** [comment]
Yellow
